src/delaunay_core/bulk_load.rs

```diff
@@ -150,10 +150,38 @@ where
     Ok(result)
 }
 
+/// Returns the [`ConstrainedDelaunayTriangulation`].
+///
+/// Conflicting edges are ignored and returned.
+/// If any conflicting edge is returned, this means input is incorrect and you may want to look into fixing it.
 pub fn bulk_load_cdt<V, DE, UE, F, L>(
     elements: Vec<V>,
     mut edges: Vec<[usize; 2]>,
 ) -> Result<ConstrainedDelaunayTriangulation<V, DE, UE, F, L>, InsertionError>
+where
+    V: HasPosition,
+    DE: Default,
+    UE: Default,
+    F: Default,
+    L: HintGenerator<<V as HasPosition>::Scalar>,
+{
+    try_bulk_load_cdt(elements, edges).map(|(cdt, _)| cdt)
```

Comment:
We'll need to preserve the previous behavior of panicing when constraints intersect.

I would suggest to implement `try_bulk_load_cdt` by passing in an `on_conflict_found: FnMut([usize; 2])` that will either panic or add this to the result vector.

Hopefully this makes sense, please ask otherwise!

Reply:
Yep, let's keep this PR focussed on the "do not split but also do not panic" scenario